package.json

```diff
@@ -27,21 +27,20 @@
   "dependencies": {
     "axios": "^0.20.0",
     "chai": "^4.2.0",
-    "dotenv": "^8.2.0",
     "cherio": "^1.0.0-rc.2",
     "diff": "^4.0.2",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "html-to-text": "^5.1.1",
     "mocha": "^8.1.3",
-    "mongoose": "^5.10.5"
-  },
-  "devDependencies": {
-    "chalk": "^4.1.0",
-    "csv-parse": "^4.12.0",
```

Comment:
@yahoo-che these are needed by the import script. That's why they're in the `devDependencies`.